### PR TITLE
[EDCO-5] HowTo component

### PR DESCRIPTION
**Pull request by Carminepo2**
## Short description
This PR adds the `HowTo` component from the [figma](https://www.figma.com/file/6Hn7Wxmvg6eCSqsMrIP7VU/UI-Components-%26-Templates?node-id=1-3895&t=OMNYFqel1ViprDJY-0).

## List of changes proposed in this pull request
- Added `HowTo` component;
- Added `assets/images` folder.

## How to test
storybook's HowTo Component stories

### CODE REVIEW

File: src/components/HowTo.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+interface Step {
+  Icon?: SvgIconComponent;
+  title: string;
+  description: string;
+}
+export interface HowToProps {
+  title: string;
+  /** Must be an array of 4 steps */
+  steps: [Step, Step, Step, Step];
```
[the remark this answers is not in the document]
Comment (stratoivandiluccio) [reply src/components/HowTo.tsx:16]: I think that this is a question for the designers. Probably if I had to deal with a similar problem, I would provide properties to control the number of steps in a row, and the alignment.

Something like that

`
{
  rowMaxSteps: number // default 4
  stepsAlignments: 'right' | 'center' | 'left' // default 'center' 
}
`

in this way we can manage different layouts. Of course this is true only above a certain breakpoint (not on a mobile for example)

[the remark this answers is not in the document]
Comment (Carminepo2) [reply src/components/HowTo.tsx:16]: Great suggestion!

File: src/components/HowTo/HowTo.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  stepsAlignment = 'center',
+}) => {
+  const isDarkTheme = theme === 'dark';
+  const background = isDarkTheme ? howToBackgroundDark : howToBackgroundLight;
+
+  const alignment = { center: 'center', left: 'start', right: 'end' }[
+    stepsAlignment
+  ];
+
+  console.time();
```
Comment (vito80ba) [inline src/components/HowTo/HowTo.tsx:48]: Is it necessary this console.time & console.timeEnd?

Comment (Carminepo2) [reply src/components/HowTo/HowTo.tsx:48]: Ups, forgot to remove it

**Review by vito80ba — APPROVED** [review]

**Review by stratoivandiluccio — APPROVED** [review]

**Pull request merged**